sim/core/consumes.go

```diff
@@ -640,28 +600,39 @@ func makePotionActivationInternal(potionType proto.Potions, character *Character
 				},
 			}),
 		}
-	} else if potionType == proto.Potions_SuperManaPotion {
-		alchStoneEquipped := character.HasAlchStone()
-		actionID := ActionID{ItemID: 22832}
+	} else if potionType == proto.Potions_MightyRejuvenationPotion {
+		actionID := ActionID{ItemID: 57193}
+		// No specific aura stats provided; adjust as needed
 		manaMetrics := character.NewManaMetrics(actionID)
+		healthMetrics := character.NewHealthMetrics(actionID)
 		return MajorCooldown{
-			Type: CooldownTypeMana,
-			ShouldActivate: func(sim *Simulation, character *Character) bool {
-				// Only pop if we have less than the max mana provided by the potion minus 1mp5 tick.
-				totalRegen := character.ManaRegenPerSecondWhileCasting() * 5
-				return character.MaxMana()-(character.CurrentMana()+totalRegen) >= 3000
-			},
```

Comment:
I think you do want the rejuv potion to still have the same ShouldActivate or you're gonna be wasting the mana gain from i

Reply:
I figured that we should allow this to be handled in the APL? In a tank sim for a prot pala or whatever, it might not be used primarily for mana at all

Reply:
Ah okay